src/content/deployment/subctl/_index.en.md

```diff
@@ -196,24 +196,38 @@ that this verification is disruptive.
 
 #### benchmark throughput
 
-`subctl benchmark throughput <kubeconfig1> <kubeconfig2>`
+`subctl benchmark throughput <kubeconfig1> [<kubeconfig2>] [flags]`
 
 The `benchmark throughput` command runs a throughput benchmark test between two specified clusters.
 When running `benchmark throughput` command, two type of tests will be issued:
 
 * Pod to Pod - where both Pods are scheduled on Gateway nodes
 * Pod to Pod - where both Pods are scheduled on non-Gateway nodes
 
+When using the flag `--intra-cluster-baseline`, the following test will be issued:
+
+* Pod to Pod - where both Pods are inside the same cluster, from Non-Gateway to Gateway
+
 #### benchmark latency
 
-`subctl benchmark latency <kubeconfig1> <kubeconfig2>`
+`subctl benchmark latency <kubeconfig1> [<kubeconfig2>] [flags]`
 
 The `benchmark latency` command runs a latency benchmark test between two specified clusters.
 When running `benchmark latency` command, two type of tests will be issued:
 
 * Pod to Pod - where both Pods are scheduled on Gateway nodes
 * Pod to Pod - where both Pods are scheduled on non-Gateway nodes
 
+When using the flag `--intra-cluster-baseline`, the following test will be issued:
+
+* Pod to Pod - where both Pods are inside the same cluster, from Non-Gateway to Gateway
+
+#### benchmark flags
+
+| Flag                                | Description
+|:------------------------------------|:---------------------------------------------------------------------------------------|
+| `--intra-cluster-baseline`          | Issue a benchmark test inside a single cluster (from Non-Gateway node to Gateway node)
```

Comment:
```suggestion
| `--intra-cluster-baseline`          | Performs the benchmark test within a single cluster between Pods from a Non-Gateway node to a Gateway node
```

Reply:
This is beyond the scope of this PR but why don't imply intra-cluster when only one kubeconfig is specified w/o the explicit `--intra-cluster-baseline` flag?

Reply:
I share the same opinion as @tpantelis. intra-cluster looks short and succinct.

Reply:
yeah we could shorten the flag to `intra-cluster` but I'm  suggesting to eliminate it.

Reply:
mm.. I guess @maayanf24 wanted to make it explicit when the user wants to test intra-cluster.

Reply:
I suppose - if the user intended inter-cluster but mistakenly specified only one kubeconfig it will error out.  I do think we should drop the `baseline` term - I'll push a PR for that.

Reply:
PR was mergeed.